src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticateResult.cs

```diff
@@ -31,12 +31,17 @@ public class AuthenticateResult
         /// <summary>
         /// Additional state values for the authentication session.
         /// </summary>
-        public AuthenticationProperties Properties => Ticket?.Properties;
+        public AuthenticationProperties Properties => Ticket?.Properties ?? Error?.Properties;
```

Comment:
There will never be both a Ticket and an Error, and the Properties collection is actually the same either way.